swift/src/IceImpl/include/Properties.h

```diff
@@ -4,29 +4,28 @@
 NS_ASSUME_NONNULL_BEGIN
 
 ICEIMPL_API @interface ICEProperties : ICELocalObject
-- (nonnull NSString*)getProperty:(NSString*)key;
-- (nullable NSString*)getIceProperty:(NSString*)key error:(NSError**)error NS_SWIFT_NAME(getIceProperty(_:));
-- (nonnull NSString*)getPropertyWithDefault:(NSString*)key value:(NSString*)value;
+- (NSString*)getProperty:(NSString*)key;
```

Comment:
is this change because we have NS_ASSUME_NONNULL_BEGIN, and all the operations return nonnull results?

Reply:
Yes